lanocrm/src/components/Layout/TopMenu/TopMenu.module.css

```
@@ -221,88 +299,269 @@
+  .ctaButton {
+    display: inline-flex;
+    position: static;
+    transform: none;
+    margin-left: 8px;
+    margin-right:8px;
+    display: inline-flex;
+  }
+
+  .ctaButton:hover {
+    transform: none;
+    box-shadow: 0 3px 8px rgba(0, 0, 0, 0.18);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)